swagger-java-client/src/main/java/io/swagger/client/api/ContainersApi.java

```diff
@@ -254,6 +254,48 @@ public Container getRegisteredContainerByPath (String repository) throws ApiExce
     return apiClient.invokeAPI(path, "GET", queryParams, postBody, headerParams, formParams, accept, contentType, authNames, returnType);
     
   }
+
```

Comment:
This is an auto-generated class and should not be modified this way. Refer to https://github.com/ga4gh/dockstore/blob/feature/implement_individual_refresh_for_IT/README.md#swagger-java-client-for-dockstore 

Since you're hitting a pre-existing web resource, I suspect that this class is out of data and needs to be regenerated.